upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1384,5 +1385,40 @@ func (n *nodeUpConfigBuilder) BuildConfig(ig *kops.InstanceGroup, apiserverAddit
 	config.Channels = n.channels
 	config.EtcdManifests = n.etcdManifests[role]
 
+	if cluster.Spec.Containerd == nil {
+		cluster.Spec.Containerd = &kops.ContainerdConfig{}
+	}
+	config.ContainerdConfig = buildContainerdConfig(cluster)
```

Comment:
One last suggestion. This only makes sense to be done if container runtime is containerd.
/lgtm
```suggestion
	if cluster.ContainerRuntime == "containerd" {
		if cluster.Spec.Containerd == nil {
			cluster.Spec.Containerd = &kops.ContainerdConfig{}
		}
		config.ContainerdConfig = buildContainerdConfig(cluster)
	}
```